dhcp/Vagrantfile

```diff
@@ -2,22 +2,22 @@
 # vi: set ft=ruby :
 
 load "../settings.sh"
-IP = NETWORK+".1"
+NETWORK = LAN_IP.split(/\.([^.]*)$/)[0]
+IP = NETWORK + ".1"
 
 Vagrant.configure("2") do |config|
   
-    config.vm.box = "fujimakishouten/debian-buster64"
-    config.vm.network "public_network", ip: IP, netmask: "255.255.255.0",
-            bridge: INTERFACE
+   config.vm.box = "VB_IMAGE"
```

Comment:
This image does not need to change. It can be the same that is used for the bionic branch. We have discussed this previously but you always forget things.

Reply:
Yes. I had to include this because there is a new issue.

https://app.vagrantup.com/fujimakishouten/boxes/debian-buster64
The Debian buster box that we are using is updated. And it is not an official box from the Debian community. It is made of one of the contributors. That's why it has many issues. One of them we provided a workaround using `vagrant-vbguest`.

Now they updated their boxes a lot. They finally found a fix to this issue and a couple of other issues. The installation goes smoothly and I am able to boot from client as well. But on the client I get a `kernel panic not syncing`. I check my `install.sh` there are no issues with it. Also asked about this to ltsp irc people. 

Then I switched to offical [debian/stretch](https://app.vagrantup.com/debian/boxes/stretch64) box. It worked and installation went smoothly and I am able to boot from client as well.

So I thought of adding it. So that we can change it later when debian community releases their offical debian buster vagrant box.

Reply:
We are talking for a DHCP/gateway box that is used only for testing.